config/manifests/bases/argocd-operator.clusterserviceversion.yaml

```diff
@@ -660,5 +660,5 @@ spec:
   maturity: alpha
   provider:
     name: Argo CD Community
-  replaces: argocd-operator.v0.1.0
-  version: 0.2.0
+  replaces: argocd-operator.v0.2.0
```

Comment:
Going forward, we may consider using `skipRanges` in the Argo CD Operator's CSV as well?